.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -24,17 +24,34 @@ body:
       label: Which component(s) of AVD impacted
       multiple: true
       options:
-        - eos_designs
+        - anta_runner
+        - cv_deploy
         - eos_cli_config_gen
-        - cvp_configlet_upload
-        - dhcp_provisioner
         - eos_config_deploy_cvp
-        - eos_config_deploy_eapi
-        - eos_snapshot
+        - eos_designs
         - eos_validate_state
-        - new components
+        - PyAVD
         - others
 
+  - type: textarea
+    id: versions
+    attributes:
+      label: Component versions
+      description: Please provide the AVD, Ansible, and Python version. e.g provide pip freeze output.
```

Comment:
I don't think a pip freeze will be very useful in most cases. Just AVD, Ansible and maybe Python version. Pip freeze would only give two of those, but clutter the issue a lot.
I would rather have 3 small inputs where you can write the versions.

Reply:
AVD version should be required. The others optional.

Reply:
addressed